package.json

```diff
@@ -10,22 +10,22 @@
     "format": "prettier --write '**/*' --ignore-unknown"
   },
   "dependencies": {
-    "@heroicons/react": "^1.0.6",
-    "@layer0/rum": "^3.0.0",
-    "classnames": "^2.3.1",
-    "next": "12.1.0",
-    "node-fetch": "^3.2.4",
+    "next": "^12.2.0",
     "react": "^18.0.1",
     "react-dom": "^18.0.1"
   },
   "devDependencies": {
-    "@layer0/cli": "^4.15.6",
-    "@layer0/core": "^4.15.6",
-    "@layer0/devtools": "^4.15.6",
-    "@layer0/next": "^4.15.6",
-    "@layer0/prefetch": "^4.15.6",
-    "@layer0/react": "^4.15.6",
+    "@heroicons/react": "^1.0.6",
+    "@layer0/cli": "^4.15.7",
+    "@layer0/core": "^4.15.7",
+    "@layer0/devtools": "^4.15.7",
+    "@layer0/next": "^4.15.7",
+    "@layer0/prefetch": "^4.15.7",
+    "@layer0/react": "^4.15.7",
+    "@layer0/rum": "^3.1.0",
```

Comment:
This is used in `_app.js` shouldn't it be a full dependency?

Reply:
Nope, that is compiled during the build time.

Even if that required that library, layer0/build.js would smartly include all dependencies that would be required in runtime.